HISTORY.rst

```diff
@@ -17,6 +17,10 @@ History
 * Building unit tests for different `Subsample` and `BlockBooststrap` instances
 * Change the sign of C_k in the `Kolmogorov-Smirnov` test documentation
 * Building a training set with a fraction between 0 and 1 with `n_samples` attribute when using `split` method from `Subsample` class.
+* Add `SplitCPRegressor`, bsaed on new `SplitCP` abstract class, to support the new CCP method
```

Comment:
typo with `bsaed` instead if `based` :-)

Reply:
Thank you !